ansul/journal.md

```diff
@@ -0,0 +1,19 @@
+# ansul90 Log File | MOC-ANDROID
+
+### 1/6/17
+
+Today learnt about layouts
```

Comment:
like @vramana suggested your journal should be in this format: if any questions ask on gitter chatroom
https://raw.githubusercontent.com/fcc-hyd/moc-android/master/sampleJournal.md